tools/skaffold-3orgs.yaml

```diff
@@ -235,3 +237,12 @@ deploy:
           enrollments.creds[1].options: "--id.type peer"
           enrollments.csrHost: network-org-3-peer-1-hlf-peer.org-3
           toolbox.enabled: true
+          hooks.serviceAccount: substra-delete-hook
+    flags:
+      install: ["--create-namespace"]
+  kubectl:
+    manifests:
+    - k8s/serviceAccount-orderer.yaml
+    - k8s/serviceAccount-org-1.yaml
+    - k8s/serviceAccount-org-2.yaml
+    - tools/k8s/serviceAccount-org-3.yaml
```

Comment:
typo

Reply:
oh no, it's not a typo.

Any reason to put org-3 in a different folder?

Reply:
I thought it was better as org-3 is only part of the skaffold configuration files in `tools/` which are not heavily maintained or used as stated in the file `tools/README.md`
I could move all of the SA in the same place if you think it is easier to maintain

Reply:
That makes sense, no strong opinion here so don't bother